torch/csrc/distributed/c10d/init.cpp

```diff
@@ -708,7 +708,18 @@ They are used in specifying strategies for reduction collectives, e.g.,
               -> std::shared_ptr<jit::PythonFutureWrapper> {
             return std::make_shared<jit::PythonFutureWrapper>(work.getFuture());
           },
-          py::call_guard<py::gil_scoped_release>());
+          py::call_guard<py::gil_scoped_release>(),
+          R"(
+            ``get_future`` retrieves a future associated with the completion of
+            ``c10d.ProcessGroup.work``. As an example, a future object can be set
+            by `future_work = dist.allreduce(bucket.get_tensors()).get_tensors()`
```

Comment:
The last part in this line should be `get_future` instead of `get_tensors`? Also, in the example just use something like `dist.allreduce(tensor)`, no need to mention GradBucket here since there is a generic API for work.